scripts/periphery.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+eval "$($HOME/.local/bin/mise activate -C $SRCROOT bash --shims)"
+
+tuist install
+tuist generate -n
+
+
+# Check if the "--full-dir" argument is provided
+if [[ "$1" == "--full-dir" ]]; then
+    full_dir=true
+else
+    full_dir=false
+fi
+
+# Perform the periphery scan and handle directory removal based on argument
+if "$full_dir"; then
```

Comment:
Defining a bool just to if else on it, we could probably put everything in the first parameter check.

Reply:
yup totally agree